src/qp/metrics/array_metrics.py

```diff
@@ -56,6 +64,45 @@ def quick_kld(p_eval, q_eval, dx=0.01):
     Dpq = dx * np.sum(p_eval * logquotient, axis=-1)
     return Dpq
 
+def quick_kolmogorov_smirnov(p_cdf, q_cdf, **kwargs):
+    """Calculate the Kolmogorov-Smirnov statistic using scipy.stats.kstest for each pair of distributions
+    in two input Ensembles. For more details see:
+    https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.kstest.html
+
+    Args:
+        p np.array of floats: A gridded array representing the CDF of a given distribution
+        q np.array of floats: A second gridded array representing the CDF of a given distribution
+
+    Returns:
+        output [KstestResult]: A array of named 2-tuples.
+        For details see: https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.kstest.html
+    """
+
+    return stats.kstest(p_cdf, q_cdf, **kwargs)
+
+def quick_moment(p_eval, grid_to_N, dx):
+    """
+    Calculates a moment of an evaluated PDF
+
+    Parameters
+    ----------
+    p_eval: numpy.ndarray, float
+        the values of a probability distribution
+    grid: numpy.ndarray, float
+        the grid upon which p_eval was evaluated
+    dx: float
+        the difference between regular grid points
+    N: int
+        order of the moment to be calculated
+
+    Returns
```

Comment:
How important is consistency in comment styling (ie, having "Returns:" in some functions, and "Returns\n_______" in others)?

Reply:
Thanks, that's a good question. I've been using an extension for VSCode called [autoDocstring](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) that will create a template with blanks to fill in that theoretically will work well with [Sphinx](https://www.sphinx-doc.org/en/master/), a tool that will automatically generate nice looking python html docs (like [this one for qp](https://qp.readthedocs.io/en/latest/qp.html))

So, long story short - I'm not really sure how important it is. I'll take a look and make sure that the format that I'm using will play nicely.

Reply:
Yeah, you're right. These docstrings are not correctly converted into the readthedocs format. Or at least they don't look very good. I'll go through and update them with the correct format. Thanks for catching this!

Reply:
Yes, readthedocs is expecting the numpy convention for formatting.

Reply:
I've just pushed a commit to address that formatting. Thanks all.